server/controllers/apiController.ts

```diff
@@ -2,10 +2,12 @@ import { Response } from 'express'
 import config from '../config'
 import HomepageService from '../services/homepageService'
 import { getTasks } from '../data/dpsServicesDataStore'
+import { StaffRole } from '../data/interfaces/staffRole'
 
 /**
  * Parse requests for API routes and orchestrate response
  */
+// TODO: ActiveCaseLoadId in here
```

Comment:
Is there something left to action for this TODO or can it be removed?

Reply:
It can be removed you're right 🎉